repos/system_upgrade/el7toel8/files/rhel_upgrade.py

```diff
@@ -44,9 +44,9 @@ def _process_packages(self, pkg_set, op):
             except dnf.exceptions.MarkingError:
                 pkgs_notfound.append(pkg_spec)
         if pkgs_notfound:
-            err_str = ('Packages marked by Leapp for installation/removal/upgrade not found '
-                       'in repository metadata: ') + ' '.join(pkgs_notfound)
-            raise dnf.exceptions.MarkingError(err_str)
+            err_str = ('Packages marked by Leapp for {} not found '
+                       'in repositories metadata: '.format(op.__name__) + ' '.join(pkgs_notfound))
+            print('Warning: ' + err_str, file=sys.stderr)
```

Comment:
@j-mracek Any idea how to achieve printing the warnings either at the beginning or at the end? Currently it is put randomly in the middle of the progress. Thx...

Reply:
What about to use stdout.flush()?